src/clients/FlexbaseClient.Card.ts

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { Card } from '../models/Card/Card';
+import { Address } from '../models/Address/Address';
+import { CardOptions } from '../models/Card/CardOptions';
+import { FlexbaseClientBase } from './FlexbaseClient.Base';
+import { FlexbaseResponse } from '../models/FlexbaseResponse';
+
+interface QueryParameters {
+    searchTerm?: string;
+    status?: string;
+}
+
+interface CardsResponse extends FlexbaseResponse{
+    cards: Card[];
+}
+
+interface CardResponse extends FlexbaseResponse{
+    card: Card;
+}
+
+interface IssueCardForm {
+    cardType: string,
+    shipTo?: Address,
+    service?: string,
+}
+
+interface UpdateCardForm {
+    expensesTypes: {
+        amount: number,
+        groups: any,
+    };
+    notifyUse: boolean;
+    shipTo?: Address,
+    creditLimit: number,
+    cardName?: string,
+}
+
+export class FlexbaseClientCard extends FlexbaseClientBase {
+    private params({ searchTerm, status }: CardOptions = {}) {
+        const params: any = {};
+
+        if (typeof searchTerm === 'string') {
+            params.searchTerm = searchTerm;
+          }
+      
+          if (typeof status === 'string') {
+            params.status = status;
+          }
+
+        return params;
+    }
+
+    async getCompanyCards({ searchTerm, status }: QueryParameters = {}): Promise<Card[] | null> {
```

Comment:
What about `getCardsByCompany`? @kyle-b2 @alegoos

Reply:
Go it